### ISSUE TITLE
Jabo_Dsound.dll fails to intialise

### ISSUE THREAD

**Issue by TheGent** [pj64team/Project64-Legacy#112]
![Jabo Audio](https://github.com/pj64team/Project64-Legacy/assets/82123790/48376496-8440-45a0-b644-a0a5a7eb3359)

Jabo_Dsound.dll fails to intialise using Windows 7 64 Bit

**Comment by Squall-Leonhart** [comment]
[Requires VS2003 redist.](https://www.microsoft.com/en-eg/download/details.aspx?id=6818)

**Comment by TheGent** [comment]
Hmm OK thank you. I will have to check my other machines for this

**Comment by Squall-Leonhart** [comment]
It is technically installed by the Dot Net 1.1 installer, but 1.0 and 1.1 are install blocked on 7+ unless using a custom MSI

**Comment by Fanatic-64** [comment]
I have a Windows XP VM with .Net 1.1 installed, I can look for the msvcp71.dll file.

**Comment by TheGent** [comment]
I will also dig out my xp box just incase

**Comment by TheGent** [comment]
Okay it seems this was the result of my PC dying and it slowly removing everything from the system. I have just tried it on Vista and there is no issue, also on another Windows 7 desktop and there is no issue. I am now installing Windows 10 on the main computer and seeing if that resolves the issue on that as well.

**Comment by TheGent** [comment]
Hmm, seems to be happening on Windows10 also. Anyone else?

**Comment by TheGent** [comment]
Ok problem solved: Had to perform a comment out in Plugin.c on 

Line 107 //if (AiRomOpen == NULL) { return FALSE; }

**Comment by TheGent** [comment]
Solved with  Plugin.c on Line 107 //if (AiRomOpen == NULL) { return FALSE; }

**Comment by Squall-Leonhart** [comment]
thats not a fix, all you're doing is masking plugin load failures.

**Comment by TheGent** [comment]
I never said it was a fix, i said it was solved by.
Obviously this needs a permanent solution, but for now so Jabo sound can be used, it bypasses.

**Comment by Azimer** [comment]
Is this a problem with XP support or gent's failing computer support?  I've loaded up PJ64 1.6 many times on fresh XPSP3 builds and never had this problem.  I do not guarantee some other software I installed didn't already install the runtime.  I agree with squall - Taking out a check to ensure the plugin is in compliance with spec is not the solution if there's an alternate solution such as bundling the redist.
